scripts/deploy-passes.js

```diff
@@ -48,10 +48,8 @@ async function main() {
   assert.isTrue(validator.length === 42)
   // assert.isTrue(operator.length === 42)
 
-  const maxTokenId = process.env.MAX_TOKEN_ID || 888
```

Comment:
why use ||?

Reply:
Because during testing I deploy much less token

Reply:
In some case you want to force a different size. But it is obsolete, now it is fixed
I will remove it